distribution/std-lib/Base/src/Time.enso

```diff
@@ -0,0 +1,23 @@
+from Base import all
```

Comment:
This file seems unnecessary. Just use the main `Time` type as entry point.
Also, the helpers here are useless. `epoch_time` should just stay as `Time . now . to_epoch_seconds`, same with `epoch_millis`. We already have `nano_time` exposed on the `Runtime` type and that is good, because it really is a low-level util for very specific uses, not something the user should stumble upon when looking for time-related-stuff. E.g. JS exposes it as `performance.now()`, for that exact reason.